visits-service-new/src/main/java/com/petclinic/visits/visitsservicenew/DataLayer/DataSetupService.java

```diff
@@ -1,27 +1,33 @@
 package com.petclinic.visits.visitsservicenew.DataLayer;
 
+import com.petclinic.visits.visitsservicenew.DomainClientLayer.VetDTO;
```

Comment:
Just wondering, but is this import necessary for your pr? You have just added it and GitHub says it's unused, I just wanna know what you're going to use it for
Thank You

Reply:
Didn't notice, thank you